src/lib/blake2/blake2.ml

```diff
@@ -2,12 +2,34 @@ open Core_kernel
 
 let digest_size_in_bits = 256
 
-let digest_size_in_bytes = 256 / 8
+let digest_size_in_bytes = digest_size_in_bits / 8
 
-include Digestif.Make_BLAKE2S (struct
+module T = Digestif.Make_BLAKE2S (struct
   let digest_size = digest_size_in_bytes
 end)
 
+include T
+
+module Stable = struct
+  module V1 = struct
+    module T = struct
+      type t = T.t [@@deriving version {asserted; unnumbered}]
+
+      include Binable.Of_stringable (struct
```

Comment:
Is this performant enough? I think we’ll want to do binio on the raw bytes of the blake2 hash to prevent string round tripping